ci-operator/templates/openshift/installer/cluster-launch-installer-libvirt-e2e.yaml

```diff
@@ -222,6 +222,9 @@ objects:
         # gcloud compute will use this key rather than create a new one
         cp -a /tmp/cluster/ssh-privatekey ~/.ssh/google_compute_engine
         cp -a /tmp/cluster/ssh-publickey ~/.ssh/google_compute_engine.pub
+        echo 'ServerAliveInterval 30' | tee -a "${HOME}"/.ssh/config
+        echo 'ServerAliveCountMax 1200' | tee -a "${HOME}"/.ssh/config
+        chmod 0600 "${HOME}"/.ssh/config
```

Comment:
This seems unrelated to what the commit log is saying.

Reply:
@cfergeau yes this should be in separate commit since without this change the server timeout happen.

Reply:
ah..yes ..good point...i'll put it in a separate commit